test/index.js

```diff
@@ -51,13 +49,11 @@ describe('BIP39', function() {
 
   describe('generateMnemonic', function() {
     vectors.english.forEach(function(v, i) {
-      it('works for tests vector ' + i, sinon.test(function() {
-        this.mock(crypto).expects('randomBytes')
-          .exactly(1)
-          .onCall(0).returns(new Buffer(v[0], 'hex'))
+      it('works for tests vector ' + i, function() {
+        function rng() { return new Buffer(v[0], 'hex') }
 
-        assert.equal(BIP39.generateMnemonic(), v[1])
-      }))
+        assert.equal(BIP39.generateMnemonic(undefined, rng), v[1])
```

Comment:
@dcousens I need the test back before I can merge, tag & release. If you don't have a chance to do it I can do it over the weekend.